src/main/java/net/tropicraft/core/client/TropicraftRenderUtils.java

```diff
@@ -80,7 +83,32 @@ public int colorMultiplier(IBlockState state, IBlockAccess world, BlockPos pos,
     public static final IItemColor BLOCK_ITEM_COLORING = new IItemColor() {
         @Override
         public int getColorFromItemstack(ItemStack stack, int tintIndex) {
-            return SandColors.getColor(tintIndex - 1);
-        }
-    };
+			return SandColors.getColor(tintIndex - 1);
+		}
+	};
+
+	public static void renderItem(ItemStack stack, float scale) {
+		if (stack != null) {
+
+			GlStateManager.pushMatrix();
+			{
```

Comment:
random brackets are random

Reply:
Not random, it's just my convention to add a scope inside matrix stack manipulation, so that it's clear how deep into the matrix stack you are.

Reply:
well-played, sir :nerd_face: